src/tests/JIT/JIT_others.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <MergedWrapperProjectReference Include="*/**/*.??proj" />
+
+    <MergedWrapperProjectReference Remove="*/**/*_d.??proj" />
+    <MergedWrapperProjectReference Remove="*/**/*_do.??proj" />
+    <MergedWrapperProjectReference Remove="*/**/*_r.??proj" />
+    <MergedWrapperProjectReference Remove="*/**/*_ro.??proj" />
+  </ItemGroup>
+
+  <Import Project="JIT_merged.targets" />
+</Project>
```

Comment:
I'm a bit concerned that this one especially (but also JIT_r, etc.) will be slow due to processing all the projects multiple times.  However, it's quite possible that this is overreacting since it is "just" the project names.

Reply:
Hmm, I think there should be some caching happening on the OS side, so far I haven't observed any huge pauses in the build because of this but I'll double-check in the next run, thanks for the heads-up.

Reply:
According to my observations this operation takes a few seconds at most, far less than the actual build times, so for now I treat it as just a nice-to-have for a future round of infra improvements.